packages/firestore/firestore.rules

```diff
@@ -172,7 +173,7 @@ service cloud.firestore {
           checkOptionalValidDate(request.resource.data, "certificateExpiration") &&
           checkOptionalValidDate(request.resource.data, "covidCertificateReleaseDate") &&
           // Check category (allows updates even if "adults")
-          request.resource.data.category in getAllCategories() &&
+          request.resource.data.category.hasAny(getAllCategories()) &&
```

Comment:
Here you should use `hasOnly`. A case we want to prevent: `customer.data.category = ["course", "non-existing-category"]`